docs/csharp/language-reference/keywords/var.md

```diff
@@ -31,7 +31,7 @@ Beginning with C# 9.0, you can use a target-typed [`new` expression](../operator
 
 ```csharp
 List<int> xs = new();
-List<int>? xy = new();
+List<int>? ys = new();
```

Comment:
This is likely my mistake, but the closing backs ticks are missing:

Reply:
Thank you for the catch @BillWagner. They are back.